package.json

```diff
@@ -25,16 +25,16 @@
     "dist/packages",
     "dist/dependencies",
```

Comment:
So this one is also broken. Last good version is:

https://unpkg.com/browse/ember-source@5.10.0-alpha.4/dist/dependencies/

If it's been broken for so long and nobody complained, I'm not sure if it needs to be brought back. There are a lot of code in the build scripts to try and support this, but clearly something isn't working. I don't care about those and did not investigate why.

If we want to remove it, it would take some effort to read through the build code to find and clean up the defunct code for this, so I'm leaving that out of scope for this PR for someone else to decide/pick up.